wazuh/wazuh-dashboard-plugins
# Statistics analysis engine dashboard queue usage is misleading

### Description

Currently, the analysis engine queue usage visualizations display not only the absolute value of the amount of decoded events, but also shows the queue usage percentages. Using the same visualization to display both values has proven to be misleading, therefore we have to design and propose a better and easier-to-understand way to show the percentages of the events queue usage.

### Issues Resolved

Closes https://github.com/wazuh/wazuh-dashboard-plugins/issues/7181

### Evidence

![image](https://github.com/user-attachments/assets/3af5c324-f8ef-4e8c-ad04-44c2151c1943)

### Test

1. Navigate to `Server management` » `Statistics`.
2. Select the `Analysis Engine` tab.
3. Check the following visualizations:
    - `Syscheck`
    - `Syscollector`
    - `Rootcheck`
    - `SCA`
    - `Host Info`
### Check List
